packages/salesforcedx-vscode-core/test/vscode-integration/orgPicker/orgList.test.ts

```diff
@@ -66,14 +66,14 @@ describe('orgList Tests', () => {
       readFileStub.callThrough();
       const orgList = new OrgList();
 
-      const authInfoObjects = await orgList.getAuthInfoObjects();
-      if (authInfoObjects) {
-        expect(authInfoObjects.length).to.equal(2);
+      const orgAuthorizations = await orgList.getOrgAuthorizations();
+      if (orgAuthorizations) {
```

Comment:
I realize this is how the test was written before, but is it clean to just assert that orgAuthorizations is defined vs the if/else
`expect(!!orgAuthorizations).to.equal(true);` note there's probably a better assertion for this

Reply:
Did some refactoring here - changes to be pushed shortly